build.gradle

```diff
@@ -28,7 +28,7 @@ dependencies {
 //	implementation('org.springframework.boot:spring-boot-starter-data-jpa')
 	implementation('org.springframework.boot:spring-boot-starter-websocket')
 //	implementation('org.flywaydb:flyway-core')
-//	runtimeOnly('org.postgresql:postgresql')
+//	runtimeOnly('org.postgresql:postgresql') 
```

Comment:
Might as well delete this space (always good to have as few files changed as possible!)